site/snippets/dynamic.ts

```diff
@@ -0,0 +1,12 @@
+import { WalletClientSigner, type SmartAccountSigner } from "@alchemy/aa-core";
+import { useDynamicContext } from "@dynamic-labs/sdk-react-core";
+
+const { primaryWallet } = useDynamicContext();
+
+const dynamicProvider = await primaryWallet?.connector?.getWeb3Provider();
+
+// a smart account signer you can use as an owner on ISmartContractAccount
+export const dynamicSigner: SmartAccountSigner = new WalletClientSigner(
+  dynamicProvider,
```

Comment:
this still needs to be wrapped in `createWalletClient`

```suggestion
  createWalletClient({ 
    transport: custom(dynamicProvider),
    chain: getChain(network!),
  }),
```

Reply:
So with Dynamic (latest), getWeb3Provider already returns a Viem WalletClient i.e. the viem interface to interact with the wallet @moldy530

Reply:
OHHH ok I missed that, that's my bad

Reply:
In fact this is a little misleading, we will get the method renamed to getWalletClient, let me update this